awstesting/integration/performance/s3GetObject/config.go

```diff
@@ -0,0 +1,198 @@
+// +build integration,perftest
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"net/http"
+	"strings"
+	"time"
+)
+
+type LogMode int
+
+func NewLogMode() *LogMode {
+	return new(LogMode)
+}
+
+func (l *LogMode) Set(v string) error {
+	switch v {
+	case "aggergated":
+		*l = AggergatedLogMode
+	case "request":
+		*l = RequestLogMode
+	default:
+		return fmt.Errorf("unknown log mode, %v", v)
+	}
+	return nil
+}
+func (l *LogMode) String() string {
+	switch *l {
+	case AggergatedLogMode:
+		return "aggergated"
```

Comment:
Nit: should be aggregated, and AggregatedLogMode

Reply:
Thanks updated. I ended up removing this LogMode all together, it wasn't actually being used, and could not be specified via a flag to the CLI parameters.